### ISSUE TITLE
PROPOSED Principle #21 "BFO class conformity"

### ISSUE THREAD

**Issue by nataled** [OBOFoundry/OBOFoundry.github.io#961]
Initial thoughts/text:

Name: BFO class conformity 

Summary:
The Basic Formal Ontology (BFO) provides a framework to enable  consistent representation of knowledge classes as either continuant (entities) or occurrent (processes).  Classifying classes within each OBO Foundry ontology according to BFO classes will provide a domain-neutral framework for ontology development. 

Purpose: 
The benefits of class conformity are: (1) to ensure that ontology content is consistently represented in their continuant and occurrent subclasses; and (2) to provide rigor on the underlying categorical structure and organization of the ontology. 

ongoing discussion points:
benefit to the community -- interoperability, orthogonality, delineated content, guidance as to how to organize an ontology
       -- should an ontology contain more than one type of BFO class ? (sometimes it is necessary
     -- 
   -- enforces reality based view so that all OBO Foundry ontologies are addressing terms in the same way.

Definitions:
Note difference between Use vs Conform to BFO (“use” = import/show directly)
continuent: A continuant is an entity that persists, endures, or continues to exist through time while maintaining its identity [Ontobee]

occurrent:  An occurrent is an entity that unfolds itself in time or it is the instantaneous boundary of such an entity (for example a beginning or an ending) or it is a temporal or spatiotemporal region which such an entity occupies_temporal_region or occupies_spatiotemporal_region. [Ontobee]

Implementation: 
     Recommendation: It is expected that OBO Foundry ontology developers will identify their pertinent BFO classes. 

Examples:
Criteria for Review:
  -- Provide an explanation of data types within the ontology and how they align to BFO classes
FMA: example 

History:
Papers implementing BFO:
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4068904/
http://www.ncbi.nlm.nih.gov/pubmed/25433798
http://www.ncbi.nlm.nih.gov/pubmed/26140188

**Comment by nlharris** [comment]
is this principle still being considered?

**Comment by nataled** [comment]
Yes. Put on back burner only because--as a brand new principle--this is a lower priority than getting already-accepted principles in line.
